model/ResultService.php

```diff
@@ -589,4 +601,44 @@ protected function updateDeliveryExecution($details, $deliveryExecution)
         return $deliveryExecution;
     }
 
-}
\ No newline at end of file
+    /**
+     * Initialize import.
+     *
+     * @param array $params
+     */
+    protected function initImport(array $params)
+    {
+        $this->report = Report::createInfo('Starting delivery executions import...');
+        $this->syncParams = $params;
+    }
+
+    /**
+     * Update report with import results.
+     *
+     * @param array $importAcknowledgment
+     */
+    protected function reportImportCompleted(array $importAcknowledgment)
+    {
+        $syncSuccess = $syncFailed = [];
+        foreach ($importAcknowledgment as $id => $data) {
```

Comment:
IMHO for array names it is nice to use plural
it increases readability.

`foreach ($importAcknowledgment as $id => $data) {`
turns into 
`foreach ($importAcknowledgments as $importAcknowledgmentId => $importAcknowledgment) {`

And accidental look to the middle of a loop immediately tells you what happens there without knowing of prehistory
`$syncSuccess[$importAcknowledgmentId] = $importAcknowledgment['deliveryId']`
istead of 
`$syncSuccess[$id] = $data['deliveryId'];`